src/constants/colors.ts

```diff
@@ -1,5 +1,6 @@
 export const LikeGreen = '#28646e';
 export const LikeCyan = '#50e3c2';
+export const LikeBlue = '#aaf1e7';
```

Comment:
Please change to `LighterCyan`

We don't have official names for colors, but I think this can be a reference
https://github.com/likecoin/likecoin-app/blob/develop/app/theme/palette.ts

Reply:
oh! didnt find it, thought its a new color haha. will add it now